### DISCUSSION TITLE
-init_desgn_config not working

### DISCUSSION THREAD

**Discussion by IngYordiDelgado** [The-OpenROAD-Project/OpenLane#1336] [General] [answered]
hi I'm trying to generate a new design in openlane i use the following command ./flow.tcl -design <design_name> -init_design_config for it, but it says the design doesn't exist, and if i create a folder with the name of the design inside the design folder, it says it have create the conf file but nothing is in there. I hope you can help me I'm just new to this.

**Comment by vijayank88** [comment, marked as answer]
@IngYordiDelgado 
Command got updated with latest OpenLane commit as follows:
```
./flow.tcl -design <design_name> -init_design_config -add_to_designs
```
Refer doc [here](https://openlane.readthedocs.io/en/latest/getting_started/quickstart.html#creating-new-designs)
